src/Microsoft.Azure.SignalR.Common/Endpoints/AadAccessKey.cs

```diff
@@ -74,36 +71,13 @@ public async Task UpdateAccessKeyAsync(IServerNameProvider provider, ILoggerFact
             {
                 return;
             }
+            await Task.Run(() => UpdateAccessKeyAsyncCore(provider, loggerFactory));
```

Comment:
we don't need to `await` the result? I think for this public method, it is only a trigger, we never expect the caller to await for the result